src/main/scala/org/broadinstitute/hail/variant/Genotype.scala

```diff
@@ -455,41 +451,83 @@ object Genotype {
     new Genotype(gt, ad, dp, gq, pl, flagFakeRef(flags))
   }
 
-  def gen(v: Variant): Gen[Genotype] = {
+  def parseGts(i: Int, j: Int): Int = {
```

Comment:
We already have this: `Genotype.gtIndex`, although it doesn't swap.  Either make it swap, or add a `Genotype.unnormalizedGTIndex`.  Parse means from strings.